src/pybind/mgr/cephadm/services/smb.py

```diff
@@ -56,6 +56,9 @@ def generate_config(
                 smb_spec, daemon_spec.daemon_id, ceph_users
             )
         )
+        config_blobs['smbmetrics_image'] = \
+            self.mgr.container_image_samba_metrics
```

Comment:
nitpick: modern python tends to use parens for line continuation not trailing backslashes. Try something like this instead:
```python
        config_blobs['smbmetrics_image'] = (
            self.mgr.container_image_samba_metrics
        )
```
(generated by running 'black' on a snippet of code)

Reply:
Interesting -- I was not aware of this modern style. Will fix.